intel_extension_for_transformers/langchain/retrievers/__init__.py

```diff
@@ -15,5 +15,5 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from .vectorstore_retriever import VectorStoreRetriever
 from .child_parent_retriever import ChildParentRetriever
+from .bge_reranker import BgeReranker
```

Comment:
can reranker apply to other embedding models except bge?

Reply:
it should work for all reranker models built with CrossEncoder structure